Utilities/InstalledSwiftPMConfiguration/Sources/exec.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+@main
+struct Exec {
+  static func main() throws {
```

Comment:
Not 100% sure we want to land this, it does seem nice to have an easy way to re-generate the correct syntax for this file if we are evolving it in the future.